app/models/govuk_publishing_components/audit_components.rb

```diff
@@ -64,7 +64,7 @@ def clean_file_name(name)
     end
 
     def get_component_name_from_full_path(path)
-      path.gsub("/_", "/")
+      path.gsub(/.*\K\/_/, "/")
```

Comment:
I'm unfamiliar with what some of this regex does. For future people, can we add in a comment to explain the regex please?

Reply:
Sure, I've added a more granular explanation to the commit message now. I assumed familiarity from when the auditing code got in https://github.com/alphagov/govuk_publishing_components/pull/1589/files#diff-83be4a6e581e7b1299afb560e4e51015R103.